model/src/main/kotlin/org/digma/intellij/plugin/ui/model/environment/EnvironmentsSupplier.kt

```diff
@@ -3,6 +3,8 @@ package org.digma.intellij.plugin.ui.model.environment
 import org.jetbrains.annotations.NotNull
 
 interface EnvironmentsSupplier {
+
+    @NotNull
     fun getEnvironments(): List<String>
     fun setCurrent(@NotNull selectedItem: String?)
```

Comment:
its out of scope of this PR
but @NotNull contradicts String? ...

Reply:
you're right. fixed.